src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Minimal.cs

```diff
@@ -159,6 +159,20 @@ private void MinimallyQualify(INamedTypeSymbol symbol)
             AddNameAndTypeArgumentsOrParameters(symbol);
         }
 
+        protected bool InObjectInitializatioContext(INamedTypeSymbol symbol)
```

Comment:
@cston, sorry for late response, we can't simplify ```NameBoundSuccessfullyToSameSymbol```, because the test ```SemanticQuickInfoSourceTests.ColorColorRangeVariable``` will fail.